### DISCUSSION TITLE
How to proceed after making changes to the code

### DISCUSSION THREAD

**Discussion by FransOv** [arendst/Tasmota#14998] [Q&A] [answered]
I have made some extensions to the support of the ADS1115 analog to digital converter. Instead of the hard-coded mode (Single ended / Differential) and voltage range this can now be set and queried via the sensor12 command. I created a fork, applied and tested the changes and created a pull request in the forked repository.
How do I proceed from there, just wait until somebody notices this and reviews and vets the changes or do I have to do something to offer the changes for review.

**Comment by Jason2866** [comment, marked as answer]
Your PR needs to be opened against this repo (branch development) to be noticed and reviewed.
